src/packages/ratmama/ratmama_parser.py

```diff
@@ -203,9 +203,26 @@ async def handle_ratmama_announcement(ctx: Context) -> None:
         platform=platform,
     )
     platform_signal = f"({rescue.platform.value.upper()}_SIGNAL)" if rescue.platform else ""
+
+    distance_str = "not found in the galaxy DB"
+    try:
+        system = await asyncio.wait_for(ctx.bot.galaxy.find_system_by_name(system_name), timeout=2)
+        if system:
+            landmark_info = await asyncio.wait_for(
+                ctx.bot.galaxy.find_nearest_landmark(system), timeout=2,
+            )
+            if landmark_info:
+                landmark, distance = landmark_info
+                if system.name != landmark.name:
+                    distance_str = f"{distance}ly from {landmark.name}"
+                else:
+                    distance_str = "landmark"
+    except asyncio.TimeoutError:
```

Comment:
i think aiohttp has a similar TimeoutError class, we should probably also catch that.